packages/conf-pkg-config/conf-pkg-config.1.1/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "1.2"
+maintainer: "unixjunkie@sdf.org"
+authors: ["Francois Berenger"]
+homepage: "http://www.freedesktop.org/wiki/Software/pkg-config/"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+dev-repo: "git://github.com/ocaml/opam-repository"
+license: "GPL"
+build: [
+  ["pkg-config" "--help"]
```

Comment:
on openbsd is there a `pkg-config` binary installed or is it named `pkgconf` ?

Reply:
The issue is that OpenBSD has a `/usr/bin/pkg-config` as part of the base system, however that version does not support `${pcfiledir}`. The version installed as the `devel/pkgconf` package _does_ support this, but gets installed only as `pkgconf`. Hence the necessity for the symlink to get OPAM packages to use the correct version.